src/components/TaskDetails/__tests__/TaskDetails.test.tsx

```diff
@@ -1,16 +1,22 @@
 import React from 'react';
 import { create } from 'react-test-renderer';
-
+import { MockedProvider } from '@apollo/react-testing';
 import { MemoryRouter } from 'react-router';
 import { ROUTES } from 'shared/routes';
+import { taskDetailsMock } from '__mocks__';
 import TaskDetails from '../TaskDetails';
 
+jest.mock('components/AppWrapper', () => 'AppWrapper');
+jest.mock('decorators/withSnackBar', () => (component: React.FC<any>) => component);
+
```

Comment:
I think those are missing:
```
jest.mock('components/NavBar', () => 'NavBar');
jest.mock('@material-ui/core/CircularProgress', () => 'CircularProgress');
```

Not sure if you are going to mock graphql request / server data for snapshot testing. If so also those should be mocked:

```
jest.mock('components/TaskCheckbox', () => 'TaskCheckbox');
jest.mock('@material-ui/core/Box', () => 'Box');
jest.mock('@material-ui/core/Typography', () => 'Typography');
```

Reply:
@michaelmoney good catch - fixed here [#a8439ac](https://github.com/BuildIt-Poland/buddy-app/pull/42/commits/a8439ac88e5d555fcf39773151447ce5cf48a25a)